docs/annotations.md

```diff
@@ -14,6 +14,7 @@ Kubernetes Ingress specification allows for annotations. We use annotations to e
 | [appgw.ingress.kubernetes.io/connection-draining-timeout](#connection-draining) | `int32` (seconds) | `30` |
 | [appgw.ingress.kubernetes.io/cookie-based-affinity](#cookie-based-affinity) | `bool` | `false` |
 | [appgw.ingress.kubernetes.io/request-timeout](#request-timeout) | `int32` (seconds) | `30` |
+| [appgw.ingress.kubernetes.io/use-private-ip](#use-private-ip) | `int32` (seconds) | `30` |
```

Comment:
The `int32 seconds` does not sound right :)
Should this be 
```
 | `bool` | `false` |
```
?

Reply:
ops!